environments/Qt/package/debiansvn/usr/share/applications/GuidoEditor.desktop

```diff
@@ -1,10 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
+Version=1.63
```

Comment:
actually the versioning scheme is different from the library versioning for the GuidoEditor since it may evolved with the same library version. Current version is 2.8 (see the GuidoEditor change log)